docs/Guide_To_The_Linux_Installers.md

```diff
@@ -0,0 +1,606 @@
+# Maintaining The Linux Installer Packaging Process
+
+This document is intended to provide a guide to the most common works needed to maintain and produce the current set of linux installer packages.
+
+## 1. Overview Of The Linux Installer Packaging Process
+
+The linux installer packaging process, currently consists of the following elements:
+
+
+  ### 1.1. Jenkins Linux Packaging Job
+  
+  This job which has restricted access. It requires several inputs that drive the packaging process:
+
+  
+  + JDK / JRE - Whether you would like the packaging process to produce the installer packages for the JDK or JRE
+  + Java version option (Which version of Java to produce packages for)
+    + Current Options : 8, 11, 17, 21, 22 
+  + Architecture - (Which hardware platform to build for)
+    + Current Options :
+      + x86_64
+      + armv7hl ( RHEL / Suse Arm 32 Platform Option )
+      + armv7l ( Debian Arm 32 Platform Option )
+      + aarch64
+      + ppc64le
+      + s390x
+      + riscv64
+      + all (This will attempt to run for all of the available architectures)
+  + Distribution - (Which Linux distributions to build for)
+    + Current Options :
+      + Alpine
+      + Debian
+      + RHEL

```

Comment:
RHEL/Fedora ?

Reply:
Yup, the top level distributions are as this, Its pretty much shorthand for RHEL based RPMs. Theres a 2nd RPM for Suse based distributions.